js/app/test/blocks_chained_events.spec.ts

```diff
@@ -91,7 +91,7 @@ test("gr.Info makes the toast show up", async ({ page }) => {
 });
 
 test("gr.Warning makes the toast show up", async ({ page }) => {
-	page.click("text=Trigger Warning");
+	await page.click("text=Trigger Warning");
```

Comment:
I think it was this @freddyaboulton. I guess we'll soon see.

Reply:
Two successful run back-to-back, so look like that might be it.

Reply:
Would explain why the info test never flakes but this one does! Thanks so much @pngwn ! Gonna merge this in to benefit other open PRs